src/unleash.ts

```diff
@@ -276,6 +277,28 @@ export class Unleash extends EventEmitter {
     return result;
   }
 
+  evaluate(name: string,
```

Comment:
new method added to our public API. This is the same is isEnabled but adds extra info about strategy evaluation result

Reply:
the intended user of this method will be proxy and frontend API